### PR TITLE
[CHARI-88] Enabled files uploading to S3 Object Storage

### PR DESCRIPTION
**Pull request by rpkyrych**

### CODE REVIEW

File: client/src/app/[locale]/(login)/sign-up/sign-up.tsx [hunk 1]
```diff
@@ -4,27 +4,32 @@ import { useTranslations } from 'next-intl';
...
+  CheckboxRadioField,
   DateField,
   FileField,
   InputField,
-  CheckboxRadioField,
-  organizationValidation,
   organizationInitialValues,
+  organizationValidation,
+  SmallBtn,
+  Title,
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(login)/sign-up/sign-up.tsx:15]: sort by length

File: client/src/s3-bucket/s3-client.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * @param folder name of the folder to upload the file
+ * @param file the content file to be uploaded
+ * @returns URL of uploaded file
+ */
+const uploadFileToBucket = async (folder: BucketFolders, file: File) => {
+  const params = {
+    Bucket: process.env.S3_BUCKET_ID,
+    Key: `${folder}/${file.name}`,
+    Body: file,
+    ACL: 'private',
```
Comment (Wolshebnik) [inline client/src/s3-bucket/s3-client.ts:32]: sort by length

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

File: client/src/app/[locale]/(login)/sign-up/sign-up.tsx (hunk 1 above)
Comment (rpkyrych) [reply client/src/app/[locale]/(login)/sign-up/sign-up.tsx:15]: All imports are formatted and optimized via WebStorm.
If there should be other formatted please add it to the project.

File: client/src/s3-bucket/s3-client.ts (hunk 2 above)
Comment (rpkyrych) [reply client/src/s3-bucket/s3-client.ts:32]: The parameters are being named according to specific guide: https://docs.digitalocean.com/reference/api/spaces-api/#hello-world-program

File: client/src/app/[locale]/(login)/sign-up/sign-up.tsx [hunk 3]
```diff
@@ -7,21 +7,22 @@ import { useTranslations } from 'next-intl';
...
   InputField,
-  showMessage,
-  CheckboxRadioField,
-  organizationValidation,
   OrganizationFormValues,
   organizationInitialValues,
+  organizationValidation,
+  showMessage,
+  SmallBtn,
+  Title,
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(login)/sign-up/sign-up.tsx:20]: sort from min to max
Скриншот 06-08-2024 105111

File: client/src/s3-bucket/s3-client.ts (hunk 2 above)
Comment (Wolshebnik) [reply client/src/s3-bucket/s3-client.ts:32]: is javascript and it is an object, and in an object it doesn't matter what order the properties are in

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

**Review by ASGladchenko — APPROVED** [review]

File: client/src/app/[locale]/(login)/sign-up/sign-up.tsx (hunk 3 above)
Comment (rpkyrych) [reply client/src/app/[locale]/(login)/sign-up/sign-up.tsx:20]: Intellij IDEA's formatter sees no other option for formatting. If you need some other option don't hesitate to add such a rule into formatter or linter.

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**